pkgs/servers/nextcloud/packages/26.json

```diff
@@ -49,6 +49,16 @@
       "agpl"
     ]
   },
+  "end_to_end_encryption": {
+    "sha256": "19vp0ggllplm84hwaqwn95122hsgfglmlk2lbwlgsjv8d36fv1wr",
+    "url": "https://github.com/nextcloud-releases/end_to_end_encryption/releases/download/v1.12.5/end_to_end_encryption-v1.12.5.tar.gz",
+    "version": "1.12.5",
```

Comment:
Why not version 1.13.1? https://github.com/nextcloud/end_to_end_encryption/releases/tag/v1.13.1

Reply:
because nextcloud 26 is still on 12.5: https://apps.nextcloud.com/apps/end_to_end_encryption

The generate.sh script added those lines automatically (See [README.md](https://github.com/NixOS/nixpkgs/tree/master/pkgs/servers/nextcloud/packages))

Reply:
Ah, okay.